src/overlay_params.cpp

```diff
@@ -538,7 +538,7 @@ parse_overlay_config(struct overlay_params *params,
    params->enabled[OVERLAY_PARAM_ENABLED_cpu_load_change] = false;
    params->enabled[OVERLAY_PARAM_ENABLED_legacy_layout] = true;
    params->enabled[OVERLAY_PARAM_ENABLED_frametime] = true;
-   params->fps_sampling_period = 500000; /* 500ms */
+   params->fps_sampling_period = 500000000; /* 500ms */
```

Comment:
This is going to change the user visible API. Also, there is no point of having so detailed value provided by user, and it is easy to miss zeros with long values.

I think it would be good to heave the fps_sampling_period at least here in us just like before.

Reply:
It stays as it is to change from config afaik

Reply:
> It stays as it is to change from config afaik
> [e19c56f#diff-011194157ef3446f8d995bb62dca19a8e0f6ed503dad901bb87498d420797f36R135](https://github.com/flightlessmango/MangoHud/commit/e19c56ff2c488c8ac3f0c6afaeb9eae11fe13503#diff-011194157ef3446f8d995bb62dca19a8e0f6ed503dad901bb87498d420797f36R135)

You are right. My bad.